### DISCUSSION TITLE
compare view different patients

### DISCUSSION THREAD

**Discussion by mafi85** [nroduit/Weasis#343] [Q&A]
Hello together,

is it possible to make an 1x2 View with images from different patients?
Actually I doesn't find any way do display it in the same time, he opens any patient in a new tab.

Any ideas?

**Comment by nroduit** [comment]
No, it is not possible nor desirable to display a series belonging to another patient than the one designated in the view.

However, it is possible to display a series on a second screen or in the same window by dragging and dropping the tabs (docking). Then you can resize the position of the tabs separation.

![image](https://user-images.githubusercontent.com/993975/196534786-ed9e2e45-75b9-4206-b2f8-277daf7bc63b.png)
